test/edu/ucsb/nceas/metacat/properties/PropertyServiceTest.java

```diff
@@ -0,0 +1,164 @@
+package edu.ucsb.nceas.metacat.properties;
+
+import edu.ucsb.nceas.LeanTestUtils;
+import edu.ucsb.nceas.utilities.GeneralPropertyException;
+import edu.ucsb.nceas.utilities.PropertyNotFoundException;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+import java.util.Arrays;
+import java.util.Map;
+import java.util.Random;
+import java.util.Set;
+import java.util.Vector;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * A JUnit test for testing Access Control in Metacat
+ */
+@RunWith(JUnit4.class)
```

Comment:
Let's discuss Junit4 versus Junit5, if you are bringing things forward from 3.

Reply:
I haven't looked into it in great depth, but I believe JUnit 5 isn't backwards-compatible with 4 or 3, so I moved to 4 as a minimum viable solution for this PR. I think there's a lot of scope to refactor and improve tests, which should really be a separate PR